docs/source/reference/configuration.md

```diff
@@ -259,6 +259,12 @@ These variables worked initially, but they were too sky130 specific and will be
 | `SPEF_WIRE_MODEL` | **Removed:** Specifies the wire model used in SPEF extraction. Options are `L` or `Pi`  <br> (Default: `L`) |
 | `SPEF_EDGE_CAP_FACTOR` | **Removed:** Specifies the edge capacitance factor used in SPEF extraction. Ranges from 0 to 1 <br> (Default: `1`) |
 
+### IR Drop Analysis
+
+|Variable|Description|
+|-|-|
+| `VSRC_LOC_FILES` | PSM loc file for power and ground nets. Variable should be provided as a json/tcl list or a space delimited tcl string as follows: `net1 file1 net2 file2`. See [this](https://github.com/The-OpenROAD-Project/OpenROAD/tree/91b99508f32d5ba1b0c284c25bd778ab40c222fe/src/psm#commands) for more info.<br> (Default: NONE)  |
```

Comment:
```suggestion
| `VSRC_LOC_FILES` | PSM loc file for power and ground nets. Variable should be provided as a json/tcl list or a space delimited tcl string as follows: `net1 file1 net2 file2`. See [this](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/psm#commands) for more info.<br> (Default: NONE)  |
```
With master branch it will tag latest path always.